src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.service.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private UserDao userDao;
+
+    @Autowired
+    public UserServiceImpl(UserDao userDao) {
```

Comment:
please either put annotation over the constructor everywhere - or put it nowhere

Reply:
ok